### PR TITLE
New Cli command that allows restarting instance

### PR DESCRIPTION
**Pull request by piotrek6641**
**What?**  
```
si seq start XX
{ id: SEQ_ID }
```
```
si inst restart SEQ_ID
```

restarted instance keeps id, input topic, output topic if set.
 

**Why?**  
`si seq start -` + `si inst kill -` + `si seq start - params` = `si inst restart -`

**Usage:**

-
-
-

**How it works:**

-
-
-

**Review checks:**

These aspects need to be checked by the reviewer:

- [ ] Verify and confirm operation (please post a screenshot) 
- [ ] All STH tests pass
- [ ] All [Scramjet Cloud Platform](https://docs.scramjet.org/platform) tests pass
- [ ] Documentation is updated or no changes

### CODE REVIEW

**Comment by piotrek6641** [comment]
Need to fix this before: https://github.com/scramjetorg/transform-hub/issues/862

**Comment by piotrek6641** [comment]
Will also need to have informations about topics in instanceInfo so they will be kept when restarting

File: packages/cli/src/lib/helpers/instance.ts [hunk 1]
```diff
@@ -23,3 +25,38 @@ export const instanceKill = async (
...
+            await getInstance(instanceId).getInfo();
+            await defer(100);
+        } catch (e:any) {
+            if (e.status === "404") {
+                ready = true;
+                break;
+            }
+        }
+    }
+    if (typeof sequenceId === "string") {
```
Comment (patuwwy) [inline packages/cli/src/lib/helpers/instance.ts:51]:
```suggestion

    if (typeof sequenceId === "string") {
```

**Comment by piotrek6641** [comment]
#644

**Review by patuwwy — APPROVED** [review]

**Review by alicja-gruzdz — APPROVED** [review]

**Review by patuwwy — APPROVED** [review]

**Pull request merged**